CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+6.0.1 / WIP
+==================
+
+Improvements added in a backwards-compatible manner:
```

Comment:
I believe that this is a minor change and it should result in new minor version. What do you think @palbecki ??

Reply:
Does API changed?

Reply:
No, the API didn't change

Reply:
So it is "substantial new functionality or improvement introduced within the private code" ?